src/Integration.UnitTests/LocalServices/ErrorListInfoBarControllerTests.cs

```diff
@@ -57,14 +55,16 @@ public class ErrorListInfoBarControllerTests
         private ConfigurableVsOutputWindowPane outputWindowPane;
         private ConfigurableConfigurationProvider configProvider;
         private ConfigurableStateManager stateManager;
+        private Mock<IKnownUIContexts> knownUIContexts;
         private ILogger logger;
 
         #region Test plumbing
 
         [TestInitialize]
         public void TestInit()
         {
-            KnownUIContextsAccessor.Reset();
+            ThreadHelper.SetCurrentThreadAsUIThread();
```

Comment:
The product code started asserting that it needed to be called on the UI thread (previously it was being invoked indirectly by the VS UI context on a different thread).